examples/optical_flow_methods_convergence.py

```diff
@@ -0,0 +1,361 @@
+    Parameters
+    ----------
+
+    input_precip: numpy array (lat, lon)
+        Input precipitation field.
+
+    motion_type : str
+        The supported motion fields are:
+
+            - linear_x: (u=2, v=0)
+            - linear_y: (u=0, v=2)
+            - rotor: rotor field
+
+    num_times: int, optional
+        Length of the observations sequence.
+
+
+    Returns
+    -------
+    synthetic_observations : numpy array
+        Sequence of observations
+    """
+
+    ideal_motion = create_motion_field(input_precip, motion_type)
+
+    # The morph function expects (lon, lat) or (x, y) dimensions.
+    # Hence, we need to swap the lat,lon axes.
+
+    # NOTE: The motion field passed to the morph function can't have any NaNs.
+    # Otherwise, it can result in a segmentation fault.
+    morphed_field, mask = morph(input_precip.swapaxes(0, 1),
+                                ideal_motion.swapaxes(1, 2))
+
+    mask = np.array(mask, dtype=bool)
+
+    synthetic_observations = np.ma.MaskedArray(morphed_field, mask=mask)
+    synthetic_observations = synthetic_observations[np.newaxis, :]
+
+    for t in range(1, num_times):
+        morphed_field, mask = morph(synthetic_observations[t - 1],
+                                    ideal_motion.swapaxes(1, 2))
+        mask = np.array(mask, dtype=bool)
+
+        morphed_field = np.ma.MaskedArray(morphed_field[np.newaxis, :],
+                                          mask=mask[np.newaxis, :])
+
+        synthetic_observations = np.ma.concatenate([synthetic_observations, morphed_field],
+                                                   axis=0)
+
+    # Swap  back to (lat, lon)
+    synthetic_observations = synthetic_observations.swapaxes(1, 2)
+
+    synthetic_observations = np.ma.masked_invalid(synthetic_observations)
+
+    synthetic_observations.data[np.ma.getmaskarray(synthetic_observations)] = 0
+
+    return ideal_motion, synthetic_observations
+
+
+def plot_optflow_method_convergence(input_precip,
+                                    optflow_method_name,
+                                    motion_type):
+    """
+    Test the convergence to the actual solution of the optical flow method used.
+
+    Parameters
+    ----------
+
+    input_precip: numpy array (lat, lon)
+        Input precipitation field.
+
+    optflow_method_name: str
+        Optical flow method name
+
+    motion_type : str
+        The supported motion fields are:
+
+            - linear_x: (u=2, v=0)
+            - linear_y: (u=0, v=2)
+            - rotor: rotor field
+    """
+
+    if optflow_method_name.lower() != "darts":
+        num_times = 2
+    else:
+        num_times = 9
+
+    ideal_motion, precip_obs = create_observations(input_precip,
+                                                   motion_type,
+                                                   num_times=num_times)
+
+    oflow_method = motion.get_method(optflow_method_name)
+
+    computed_motion = oflow_method(precip_obs, verbose=False)
+
```

Comment:
Add additional step to store precip field in mm/h so that we can plot it later.

```suggestion

precip_obs, _ = stp.utils.dB_transform(precip_obs, inverse=True)

```